c-api-examples/decode-file-c-api.c

```diff
@@ -60,13 +95,33 @@ int32_t main(int32_t argc, char *argv[]) {
   config.rule2_min_trailing_silence = 1.2;
   config.rule3_min_utterance_length = 300;
 
+  cag_option_context context;
+  char identifier;
+  const char *value;
+
+  cag_option_prepare(&context, options, CAG_ARRAY_SIZE(options), argc, argv);
+
+  while (cag_option_fetch(&context)) {
+    identifier = cag_option_get(&context);
+    value = cag_option_get_value(&context);
+    switch (identifier) {
+      case 't': config.model_config.tokens = value; break;
+      case 'e': config.model_config.encoder = value; break;
+      case 'd': config.model_config.decoder = value; break;
+      case 'j': config.model_config.joiner = value; break;
+      case 'n': config.model_config.num_threads = atoi(value); break;
+      case 'p': config.model_config.provider = value; break;
+      case 'm': config.decoding_method = value; break;
```

Comment:
Shall we add a `default` statement to handle unknown options?

Reply:
All the values are set up their defaults before reaching this `while`. So, we don't need a `default` statement. I can add a `default` statement that does nothing. Thanks.

Reply:
Added a `default` statement that does nothing. :-)